dlp/snippets/inspect/inspect_string_custom_hotword.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package inspect
+
+// [START dlp_inspect_string_custom_hotword]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	dlp "cloud.google.com/go/dlp/apiv2"
+	"cloud.google.com/go/dlp/apiv2/dlppb"
+	"google.golang.org/api/option"
+)
+
+// inspectStringCustomHotWord inspects a string from sensitive data by using a custom hot word
+func inspectStringCustomHotWord(w io.Writer, projectID, textToInspect, customHotWord, infoTypeName string) error {
+	// projectID := "my-project-id"
+	// textToInspect := "patient name: John Doe"
+	// customHotWord := "patient"
+	// infoTypeName := "PERSON_NAME"
+
+	ctx := context.Background()
+
+	// Initialize a client once and reuse it to send multiple requests. Clients
+	// are safe to use across goroutines. When the client is no longer needed,
+	// call the Close method to cleanup its resources.
+	client, err := dlp.NewRESTClient(ctx, option.WithCredentialsFile("C:/Users/aarsh.dhokai/Desktop/cred.json"))
```

Comment:
issue: use Application Default Credentials rather than manually passing in the path to your credentials file.

If using ADC is not possible, replace the actual path of your credential file with something like `"path/to/your/credentials/file.json"` or something similar.

Reply:
fixed it.